_spinnaker/armory_halyard.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: post
+title: Armory Halyard
+order: 100
+---
+## Table of Contents
+
+ * [**hal armory**](#hal-armory)
+ * [**hal armory dinghy**](#hal-armory-dinghy)
+ * [**hal armory dinghy disable**](#hal-armory-dinghy-disable)
+ * [**hal armory dinghy edit**](#hal-armory-dinghy-edit)
+ * [**hal armory dinghy enable**](#hal-armory-dinghy-enable)
+ * [**hal armory init**](#hal-armory-init)
+
```

Comment:
This section can/should be replaced with:

```
{:.no_toc}
* This is a placeholder for an unordered list that will be replaced with ToC. To exclude a header, add {:.no_toc} after it.
{:toc}
```

This will automagically generate the TOC based on the subsequent headings.